src/ethlance/server/graphql/resolvers.cljs

```diff
@@ -442,5 +512,12 @@
                     :Arbiter {:arbiter_feedback arbiter->feedback-resolver}
                     :Feedback {:feedback_toUserType feedback->to-user-type-resolver
                                :feedback_fromUserType feedback->from-user-type-resolver}
-                    :Mutation {:signIn sign-in-mutation}}
-  )
+                    :Mutation {:signIn sign-in-mutation
```

Comment:
We're missing here `createInvoice` `payInvoice` `cancelInvoice` `createJobProposal`

Reply:
Addressed